sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Generated/Models/AzureApplicationInsightsDataFeedPatch.Serialization.cs

```diff
@@ -12,137 +12,5 @@ namespace Azure.AI.MetricsAdvisor.Models
 {
     internal partial class AzureApplicationInsightsDataFeedPatch : IUtf8JsonSerializable
     {
-        void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
-        {
-            writer.WriteStartObject();
-            if (Optional.IsDefined(DataSourceParameter))
-            {
-                writer.WritePropertyName("dataSourceParameter");
-                writer.WriteObjectValue(DataSourceParameter);
-            }
-            writer.WritePropertyName("dataSourceType");
-            writer.WriteStringValue(DataSourceType.ToString());
-            if (Optional.IsDefined(DataFeedName))
-            {
-                writer.WritePropertyName("dataFeedName");
-                writer.WriteStringValue(DataFeedName);
-            }
```

Comment:
As you can see, the serialization method does not send null to the service. Unfortunately, we need to overwrite all serialization methods for Patch models (used during Update operations only) in order to send null when a property is not present.